### DISCUSSION TITLE
Improve networking documentation for Podman 5.0+

### DISCUSSION THREAD

**Discussion by wonderplayer** [containers/podman#24720] [Ideas]
Hello,
I'm new to Podman and containerization in general. I was trying to create multiple containers and see how they communicate with each other, and this was my first roadblock.
 Found documentation in [docs/tutorials/basic_networking.md](https://github.com/containers/podman/blob/main/docs/tutorials/basic_networking.md) on how to set up network, so containers can communicate with each other in rootless environment.
Due to my inexperience, I failed to notice that documentation was explaining about `slirp4netns` as default network setup, which is not true since version 5.0 according to [this](https://blog.podman.io/2024/03/podman-5-0-breaking-changes-in-detail) blog post.
Spent several hours trying to understand what I'm doing wrong and why my containers can't communicate with each other using host's IP address and the answer was so simple after I found another [blog post](https://blog.podman.io/2024/10/podman-5-3-changes-for-improved-networking-experience-with-pasta/) in Arch Linux documentation by luck.

So, would it be possible for somebody to update that documentation for the most recent Podman version, or mention that `pasta` is now default and describe how to communicate between containers with default `pasta` network setup. Believe it could be helpful to other people trying to use rootless containers in Podman.

Thanks!

**Comment by eriksjolund** [comment]
I have written some documentation
https://github.com/eriksjolund/podman-networking-docs/

**Reply by wonderplayer** [reply]
Wow, that looks quite advanced.
Maybe part of it could be added to basic network setup, where nothing complex needs to be achieved - just a regular communication between containers.
Especially in case with localhost using pods, as it looks like it works differently with pasta, if at all.
